### ISSUE TITLE
Wiki page unloading

### ISSUE THREAD

**Issue by CharonM72** [Bionus/imgbrd-grabber#1516]
**Bug description**

For some reason, the Wiki information for a tag loads properly while the page is loading, but once it loads it changes into just a text link to the tag. This appears to only happen in the initial loading of a page, but it loads properly and stays when refreshing the same page.

It may well be an issue with my configuration.

**Steps to reproduce**

1. Search a single tag that has a wiki page
2. Watch the Wiki tab

**Expected behavior**

The Wiki information should stay after loading.

**Context**

[settings.zip](https://github.com/Bionus/imgbrd-grabber/files/2636771/settings.zip)

No notable errors in main.log.

**Screenshots**

While loading:
![2018-12-02](https://user-images.githubusercontent.com/1299220/49337588-8efb1100-f5e3-11e8-851c-9f7c0b3fb514.png)
After loading:
![2018-12-02 1](https://user-images.githubusercontent.com/1299220/49337589-95898880-f5e3-11e8-8954-e7300075e0ca.png)

**System information**

- OS: Windows 10 Pro x64
- Grabber version: 6.0.6

**Comment by CharonM72** [comment]
Playing around with it, it looks like it might be loading the wiki from Sankaku? Since Sankaku's wiki is basically just a link to the tag. Also, is there any way to set which site to pull the wiki page from? Or at least show which site it's using? Some sites seem better than others.

**Comment by Bionus** [comment]
Indeed, the way it works currently if you have multiple sources supporting wikis is that the latest loaded one is shown (each source overriding the previous' wiki when loaded).

Choosing which wiki to show could be a solution, but I feel it would be annoying to change the site every time. Another solution could be to keep the longest wiki (with the most text/information). A mix would be to allow to see them all with a select box at the top, and sort them by length, showing the longest by default.

**Comment by CharonM72** [comment]
I like that last idea. Barring that, longest would be best. It's annoying when I see it load a detailed wiki, and then Sankaku's loads and overwrites it with nothing.
